.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -55,15 +53,10 @@ _Due: <release-deadline>_
 
 **Release Completion (assigned to: )**
 _Due: <release-deadline>_
-- [ ] Delete Durable Functions packages from the [ADO test feed](https://dev.azure.com/durabletaskframework/Durable%20Task%20Framework%20CI/_artifacts/feed/durabletask-test).
-- [ ] Run the [Durable Functions release pipeline](https://dev.azure.com/durabletaskframework/Durable%20Task%20Framework%20CI/_build?definitionId=23) and select `main` as the branch.
-- [ ] Add the Durable Functions package to the [ADO feed](https://dev.azure.com/durabletaskframework/Durable%20Task%20Framework%20CI/_artifacts/feed/durabletask) using [this pipeline](https://dev.azure.com/durabletaskframework/Durable%20Task%20Framework%20CI/_release?_a=releases&view=mine&definitionId=11).
+- [ ] Run the [Durable Functions release pipeline](https://azfunc.visualstudio.com/internal/_build?definitionId=673) and select `v3.x` or `dev` as the branch. Choose `dev` if you are making a v2.x relese, otherwise choose v3.x.
 - [ ] Upload the Durable Functions package to NuGet (directly to nuget.org).
 - [ ] Upload .NET Isolated worker extension package to NuGet (directly to nuget.org).
 - [ ] Create a PR in the [Azure Functions templates repo](https://github.com/Azure/azure-functions-templates) targeting branch `dev` to update all references of "Microsoft.Azure.WebJobs.Extensions.DurableTask" (search for this string in the code) to the latest version.
 - [ ] _if and only if this is a new major release_, Create a PR in the [Azure Functions bundles repo](https://github.com/Azure/azure-functions-extension-bundles) to update bundles to the latest version .
-- [ ] Merge all pending PR docs from `pending_docs.md.`
-- [ ] Reset `pending_docs.md` and `release_notes.md` in the `dev` branch. You will want to save `release_notes.md` somewhere for when you publish release notes.
-- [ ] Publish release notes from the pre-reset `release_notes.md.`
```

Comment:
I deleted these steps since we haven't used the pending_docs.md file in years. I also noticed that we don't use release_notes.md, but rather we use the "Generate release notes" functionality when creating new Release notes. I'm open to keeping these steps if folks think otherwise.

Reply:
I'm fine with the "Generate release notes functionality". Happy to remove steps where we can. :)